src/main/java/foodwhere/logic/commands/SEditCommand.java

```diff
@@ -87,7 +103,25 @@ private static Stall createEditedStall(Stall stallToEdit, EditStallDescriptor ed
         Address updatedAddress = editStallDescriptor.getAddress().orElse(stallToEdit.getAddress());
         Set<Tag> updatedTags = editStallDescriptor.getTags().orElse(stallToEdit.getTags());
 
-        return new Stall(updatedName, updatedAddress, updatedTags);
+        Set<Review> reviewsToUpdate = stallToEdit.getReviews();
```

Comment:
I think it seems a bit strange to do reviews editing outside the EditStallDescriptor object, why not edit the set of reviews inside EditStallDescriptor as well?